custom/sensors/CustomCANCoder.java

```diff
@@ -1,23 +1,25 @@
 package org.usfirst.frc4904.standard.custom.sensors;
 
-import com.ctre.phoenix.sensors.CANCoder;
-import com.ctre.phoenix.sensors.CANCoderConfiguration;
-import com.ctre.phoenix.sensors.SensorTimeBase;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+// import com.ctre.phoenix.sensors.SensorTimeBase;
+// TODO: what is sensor time base why is it gone
 
 // import org.usfirst.frc4904.standard.LogKitten;
 import org.usfirst.frc4904.standard.Util;
 
-public class CustomCANCoder extends CANCoder implements CustomEncoder {
+public class CustomCANCoder extends CANcoder implements CustomEncoder {
```

Comment:
is CANcoder one word? the C is lowercase everywhere except the class name

Reply:
for future reference, i think it's CANcoder and not CANCoder bc the ctre imports are CANcoder, but i don't want to change tho whole class name and stuff so it stays